src/styles/main.scss

```diff
@@ -0,0 +1,10 @@
+@import "_variables.scss";
+@import "clock.scss";
+
+body {
+  margin: 0;
+
+  .clock {
+    margin: auto;
+  }
+}
```

Comment:
i think you can move `.clock` selector to the `clock.scss` file, and the `body { margin: 0; }` code to the `_reset.scss` file :)

Reply:
I did not want to use the margin property for a BEM block, that is why I put .clock selector setting the element's margins within the 'body' block.